src/Codeception/Lib/Interfaces/MultiSession.php

```diff
@@ -8,7 +8,7 @@ public function _initializeSession(): void;
 
     public function _loadSession($session): void;
 
-    public function _backupSession(): array;
+    public function _backupSession();
```

Comment:
`[TypeError] Codeception\Module\WebDriver::_backupSession(): Return value must be of type array, Facebook\WebDriver\Remote\RemoteWebDriver returned`
See https://github.com/Codeception/module-webdriver/runs/5079691268

Reply:
yes, leave it untyped, `mixed` if you want so.

Reply:
I'll leave it untyped, thanks

Reply:
Could the `WebDriver::_backupSession()` not return the following instead?

```php
public function _backupSession(): array
{
    return [
        'driver' => $this->webDriver,
    ];
}
```

It would mean a breaking change for 5.0, but this would be the best time to do so. That way `_backupSession()` can have its return type fixed as `array` (which would be a sensible and portable value across implementors).